docs/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -7,8 +7,10 @@ This is especially important if there are a large number of records to sync and/
 Incremental syncs are usually implemented using a cursor value (like a timestamp) that delineates which data was pulled and which data is new. A very common cursor value is an `updated_at` timestamp. This cursor means that records whose `updated_at` value is less than or equal than that cursor value have been synced already, and that the next sync should only export records whose `updated_at` value is greater than the cursor value.
 
 To use incremental syncs, the API endpoint needs to fullfil the following requirements:
-* Records contain a date/time field that defines when this record was last updated (the "cursor field")
+* Records contain a top-level date/time field that defines when this record was last updated (the "cursor field")
+  * If the record's cursor field is nested, you can use an "Add Field" transformation to copy it to the top-level, and a Remove Field to remove it from the object. This will effectively move the field to the top-level of the record
```

Comment:
Do we have plans or an issue to allow defining the cursor field as a path so that nested fields can be accessed directly? The Add/Remove Field approach feels pretty hacky to me, but an okay workaround until we are able to support nested cursor fields natively

Reply:
Super hacky. We should design a better way to remap fields.

That being said, there's no plan to support nested cursor fields because [normalization will stop supporting them](https://docs.google.com/document/d/1T77e4V54g7-RiG02fm406iDa6PO-kkLJU0c6wJRyZhg/).